tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist =
+    py37,
+    py37-flake8
+
+[testenv]
+commands = python ./runtests.py
```

Comment:
So interestingly, this env shows as "passing" even when there are test failures. I think we should get rid of runtests.py and use pytest to run the suite. It can discover and run unittest tests just fine, as long as they use the right prefixes for test discovery, which it looks like they do.

Reply:
So that I'm clear, should I also delete `runtest.py` as well?

Reply:
correct. we don't need it since we can just run `tox` locally, which will run the test suite with pytest.

Reply:
We should also remove the reference to `runtest.py` from `setup.py`

Reply:
Done and done.